cmd/algocfg/profileCommand.go

```diff
@@ -44,6 +44,7 @@ var (
 			cfg.EnableDeveloperAPI = true
 			cfg.MaxAcctLookback = 256
 			cfg.EnableTxnEvalTracer = true
+			cfg.DisableAPIAuth = true
```

Comment:
It makes sense that with this new feature that the final version would be to default to disabling authentication for DeveloperAPI algod's. However, we need to communicate this change very loudly to the community. Unfortunately, there could be some people not following best practices and who block access to their developer endpoint via the token exclusively. If they don't look at the changelogs and upgrade their algod, they would be blindsided.

@algoanne

Reply:
I'm not so concerned with the true defaults these days, but we could set it for the "development" profile in `algocfg`

Reply:
lol, thats what you're commenting on! Good idea Shiqi!